airbyte-integrations/connectors/destination-bigquery-denormalized/src/test/java/io/airbyte/integrations/destination/bigquery/formatter/util/FormatterUtilTest.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination.bigquery.formatter.util;
+
+import static io.airbyte.integrations.destination.bigquery.formatter.util.FormatterUtil.TYPE_FIELD;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Test;
+
+class FormatterUtilTest {
+
+  private final ObjectMapper mapper = new ObjectMapper();
+
+  @Test
+  void isAirbyteArray_typeIsNull() {
+    JsonNode arrayNode = mapper.createArrayNode().add("one").add("two");
```

Comment:
I think these tests would all be easier to read if they used a literal json string:
```java
arrayNode = mapper.readTree("""["one", "two"]""");

arrayNode = mapper.readTree(
  """
  {
    "type": ["array"]
  }
  """);

// etc
```